### ISSUE TITLE
Process crashes when using libasyncProfiler

### ISSUE THREAD

**Issue by adabugra** [async-profiler/async-profiler#1091]
### Describe the bug

Process crashes when using libasyncProfiler with this start flags -> `-agentpath:/root/async-profiler-2.1-malloc-linux-x64/async-profiler-2.1-malloc-linux-x64/build/libasyncProfiler.so=start,event=nativemem,file=malloc.jfr -Xms41480M -Xmx41480M -Dcom.sun.management.jmxremote -Dcom.sun.management.jmxremote.port=1100 -Dcom.sun.management.jmxremote.ssl=false -Dcom.sun.management.jmxremote.authenticate=false -Djava.rmi.server.hostname=0.0.0.0 -XX:+UnlockDiagnosticVMOptions --add-modules=jdk.incubator.vector -XX:NativeMemoryTracking=detail -XX:+PrintNMTStatistics -Dio.netty.leakDetection.level=paranoid -XX:+UseG1GC -XX:+ParallelRefProcEnabled -XX:MaxGCPauseMillis=200 -XX:+UnlockExperimentalVMOptions -XX:+DisableExplicitGC -XX:+AlwaysPreTouch -XX:G1HeapWastePercent=5 -XX:G1MixedGCCountTarget=4 -XX:InitiatingHeapOccupancyPercent=15 -XX:G1MixedGCLiveThresholdPercent=90 -XX:G1RSetUpdatingPauseTimePercent=5 -XX:SurvivorRatio=32 -XX:+PerfDisableSharedMem -XX:MaxTenuringThreshold=1 -Dusing.aikars.flags=https://mcflags.emc.gs -Daikars.new.flags=true -XX:G1NewSizePercent=40 -XX:G1MaxNewSizePercent=50 -XX:G1HeapRegionSize=16M -XX:G1ReservePercent=15 tramontanee.jar --nogui`

### Reproduction Steps

Start the server, wait 30-40 minutes and server will crash.

### Additional Information/Context

[hs_err_pid100766.log](https://github.com/user-attachments/files/18254453/hs_err_pid100766.log)

### Async-profiler version

async-profiler-2.1-malloc-linux-x64

**Comment by adabugra** [comment]
```
root@limenet:~/async-profiler-2.1-malloc-linux-x64/async-profiler-2.1-malloc-linux-x64/build# java -cp converter.jar MallocReport malloc-latestt.jfr malloc.html
Exception in thread "main" java.lang.NullPointerException: Cannot invoke "String.hashCode()" because "<local3>" is null
        at one.jfr.JfrReader.createElement(JfrReader.java:268)
        at one.jfr.JfrReader.readElement(JfrReader.java:259)
        at one.jfr.JfrReader.readMeta(JfrReader.java:247)
        at one.jfr.JfrReader.readChunk(JfrReader.java:227)
        at one.jfr.JfrReader.<init>(JfrReader.java:84)
        at MallocReport.main(MallocReport.java:141)
```

**Comment by apangin** [comment]
`2.1-malloc` is an old experimental version, it is not supported.

Support for native memory profiling has been added only a week ago. You're welcome to use our [Nightly builds](https://github.com/async-profiler/async-profiler/releases/tag/nightly).

I also noticed you did not set sampling interval. I'd suggest to replace `event=nativemem` with `nativemem=1m` to ensure native memory profiling does not cause any noticeable overhead.
